packages/hs-react-ui/src/components/InteractionFeedback/InteractionFeedback.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useCallback, useState } from 'react';
+import { animated, useTransition } from 'react-spring';
+import styled, { StyledComponentBase } from 'styled-components';
+import shortid from 'shortid';
+import colors from '../../enums/colors';
+import useResizeObserver from 'use-resize-observer/polyfilled';
+
+const Circle = styled.svg`
+  pointer-events: none;
+  position: absolute;
+  top: 0;
+  left: 0;
+`;
+
+const Container = styled(animated.div)`
+  position: relative;
+`;
+
+type Animation = { cx: string; cy: string; id: string };
+type Transition = { r: string } & Animation;
+type InteractionFeedbackProps = {
+  StyledContainer?: string & StyledComponentBase<any, {}>;
+
+  children: React.ReactNode;
+  interpolationFunctions?: Record<string, (val: any) => any>;
+  transitionProps: any;
```

Comment:
If we're going to default these values, transitionProps can be optional too

Reply:
totes. Also added a comment saying to make types gud